include/aspect/assembly.h

```diff
@@ -197,6 +202,11 @@ namespace aspect
            */
           MaterialModel::MaterialModelInputs<dim> explicit_material_model_inputs;
           MaterialModel::MaterialModelOutputs<dim> explicit_material_model_outputs;
+
+          /**
+           * Heating model outputs computed at the current linearization point.
```

Comment:
I assume the different heating models will add things up in this variable. Can you state so here?